### PR TITLE
feat: compatible with webpack v4

### PR DESCRIPTION
**Pull request by kagawagao**
related issue #17

### CODE REVIEW

**Comment by van-nguyen** [comment]
@kagawagao Thanks for the PR, I've made a comment and also unit test is failing. If you can address these I'd be happy to merge. Thanks 👍

**Comment by kagawagao** [comment]
I'm trying to solve this problem, `Stats Data API` has changed too

**Comment by kagawagao** [comment]
There is something conflict with `webpack v4`, maybe need to refactor unit test case


so....

help wanted 😂

**Comment by kagawagao** [comment]
Although the unit test passed, there is still a problem here. if I set `webpack` config `mode` as `development`, it work, but other cases are not. I tracked the code, this plugin is not been applied into compiler, I can not find reason...

**Comment by Gisleburt** [comment]
It probably goes without saying (so apologies if I'm stating the obvious) but when this merges, don't forget to bump the major version so it doesn't mess with people on WP3. 😉

**Comment by van-nguyen** [comment]
@Gisleburt No probs, I'll make sure that happens.

I know this might be critical to folks out there, but this weekend has been quite hectic for me. I'll try to get some time this week.

**Comment by van-nguyen** [comment]
@kagawagao Managed to fix the production issue, seems like `istanbul` package was removed and it was causing problems. I will commit the fix directly to master.

File: module.js [hunk 1]
```diff
@@ -34,19 +34,35 @@ class WebpackCdnPlugin {
...
+              data.assets.css = WebpackCdnPlugin._getCss(modules, ...getArgs).concat(data.assets.css);
+            }
           }
-        }
-        callback(null, data);
+          callback(null, data);
+        });
       });
-    });
+    } else {
```
[the remark this answers is not in the document]
Comment (kagawagao) [reply module.js]: in the latest commit, this problem doesn't exist

**Pull request merged**